components/automate-ui/src/app/pages/team-management/team-management.component.spec.ts

```diff
@@ -106,7 +104,7 @@ describe('TeamManagementComponent', () => {
 
     beforeEach(() => {
       store = TestBed.get(Store);
-      component.iamMajorVersion$ = observableOf(<IAMMajorVersion>'v1');
+      component.isMajorV1 = true;
```

Comment:
🤔 I haven't surveyed the variable name we're using elsewhere, but I wonder if we should be more consistent with using `iam` or not . . .